src/test/java/com/jcabi/w3c/DefaultCssValidatorTest.java

```diff
@@ -82,4 +85,23 @@ public void ignoresEntireDocument() throws Exception {
         MatcherAssert.assertThat(response.toString(), response.valid());
     }
 
+    /**
+     * DefaultHtmlValidator throw IOException when W3C server is unavailable.
+     * @throws Exception If something goes wrong inside
+     * @todo #10:30min DefaultCssValidator have to be updated to throw only
+     *  IOException when W3C validation server is unavailable. Any other
+     *  exception type can be confusing for users. Remove @Ignore annotation
+     *  after finishing implementation.
+     */
+    @Ignore
+    @Test(expected = IOException.class)
+    public void throwsIOExceptionWhenValidationServerIsUnavailable()
+        throws Exception {
+        final MkContainer container = new MkGrizzlyContainer().next(
+            new MkAnswer.Simple(HttpURLConnection.HTTP_UNAVAILABLE)
+        ).start();
+        new DefaultCssValidator(container.home()).validate("body { }");
+        container.stop();
```

Comment:
Not possible. According to [The try-with-resources Statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html). Any object that implements `java.lang.AutoCloseable` can be used as a resource. `MkContainer` does not extends `java.lang.AutoCloseable` here.